.github/workflows/graph-develop.yaml

```diff
@@ -0,0 +1,68 @@
+name: Deploy Graph to develop
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install
+        run: yarn --frozen-lockfile
+      - name: Lint
+        run: yarn lint
+
+  generate-manifests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install
```

Comment:
We install dependencies two times! once in lint and once in generate-manifest. We can depend generate-manifest to lint
```
    needs: [lint]
```
And for others just make generate-manifest the dependency